### DISCUSSION TITLE
ETA for 22.0.4 - #22699 in release

### DISCUSSION THREAD

**Discussion by tsili852** [keycloak/keycloak#23204] [Q&A] [answered]
There is a fix commit in the nightly build that we are really looking forward to be included in a release. Notably [22699](https://github.com/keycloak/keycloak/pull/22699). 
It affects the display of custom mappers when used with LDAP.
Do you have any timeline when this will be included in a release ?
Do you have a date for the release of 22.0.4 ?
Thanks :)

**Comment by abstractj** [comment, marked as answer]
@tsili852 we don't have any eta for 22.0.4, so please keep an eye on future release announcements.
